src/channel.js

```diff
@@ -133,6 +136,14 @@ export default class Channel {
     }
   }
 
+  /**
+   * Register the channel to which events should be proxied
+   * @param {Channel} channel
+   */
+  addProxyChannel(channel) {
+    this._proxyChannels.push(channel);
```

Comment:
`channel instanceof Channel`? 
And also `this._proxyChannels.includes(channel)`